docs/manual/comparison_with_pandas.rst

```diff
@@ -0,0 +1,246 @@
+.. py:currentmodule:: datatable
+
+Comparison with Pandas
+=======================
+A lot of potential ``datatable`` users are likely to have some familiarity with `pandas <https://pandas.pydata.org/pandas-docs/stable/index.html>`__; as such, this page provides some examples of how various ``pandas`` operations can be performed within ``datatable``. ``datatable`` emphasizes speed and big data support, two areas that ``pandas`` struggles with; it also has an expressive and concise syntax, which makes ``datatable`` also useful for small datasets.
+
+.. note:: In ``pandas``, there are two fundamental data structures, `Series <https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.html#pandas.Series>`__ and `DataFrame <https://pandas.pydata.org/pandas-docs/stable/user_guide/dsintro.html#dataframe>`__. In ``datatable``, there is only one fundamental data structure - the `Frame <https://datatable.readthedocs.io/en/latest/api/frame.html#datatable-frame>`__. Most of the comparisons will be between a ``pandas`` DataFrame and ``datatable`` Frame.
+
+.. code-block:: python
+
+    import pandas as pd
+    import numpy as np
+    from datatable import dt, f, by, g, join, sort, update, ifelse
+
+    data = {"A": [1, 2, 3, 4, 5],
+            "B": [4, 5, 6, 7, 8],
+            "C": [7, 8, 9, 10, 11],
+            "D": [5, 7, 2, 9, -1]}
+
+    # datatable
+    DT = dt.Frame(data)
+
+    # pandas
+    df = pd.DataFrame(data)
+
+
+Row and Column Selection
+------------------------
+
+=================================================  ============================================
+        Pandas                                              datatable
+=================================================  ============================================
+Select a single row
+``df.loc[2]``                                        ``DT[2, :]``
+
+Select ``n`` rows
```

Comment:
How about `select several rows by their indices`, since there is no ``n`` in the formulas